torch/fx/passes/split_utils.py

```diff
@@ -274,7 +278,7 @@ def remap_func(x):
             kwargs=None,
         )
 
-        if len(outs) == 1:
+        if len(outs) == 1 and not return_tuple:
```

Comment:
Hmm, wondering why we need this extra ```not return_tuple``` condition.

Reply:
https://github.com/pytorch/pytorch/blob/main/torch/fx/graph.py#L1264

We have an implicit assumption that the return arg should be a single tuple.